### ISSUE TITLE
State of the art MobileNet v3

### ISSUE THREAD

**Issue by JulienMaille** [qubvel/segmentation_models.pytorch#267]
As anyone tried to adapt it to this library?
https://github.com/ekzhang/fastseg

I'm willing to try, if anyone can help me getting started.

According to https://news.ycombinator.com/item?id=24127499

> This work was done at Nvidia as part of a larger project that required semantic segmentation. But although MobileNetV3 has state-of-the-art performance on many image tasks like classification, detection, and segmentation - there are no public implementations of MobileNetV3 for semantic segmentation with good accuracy.
Looking through implementations on GitHub, we saw accuracies of 40-50% mIoU, which is frankly unacceptable given that the paper claims 72.6% mIoU. So over the past few months at Nvidia, I worked with some researchers from ADLR (https://nv-adlr.github.io/) to implement MobileNetV3 in PyTorch. After a bunch of hyperparameter tuning, we managed to train it to within 0.3% of the accuracy reported in the paper: https://arxiv.org/abs/1905.02244v5. See the "Metrics" section of the GitHub README for more detailed information.
